### PR TITLE
Announce AKS Kubernetes upgrade to 1.21

### PR DESCRIPTION
**Pull request by lemeurherve**
Ref: https://github.com/jenkins-infra/helpdesk/issues/2866

### CODE REVIEW

File: content/issues/2022-04-26-maintenance-AKS-k8s [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  - ldap.jenkins.io
+  - plugins.jenkins.io
+  - release.ci.jenkins.io
+  - reports.jenkins.io
+  - www.jenkins.io
+  - uplink.jenkins.io
+section: issue
+---
+
+We'll do an AKS clusters upgrade from 1.20 to 1.21 this Tuesday, 26th of April 2022, starting at 11:00 UTC.
```
Comment (lemeurherve) [inline content/issues/2022-04-26-maintenance-AKS-k8s:26]:
```suggestion
date: 2022-04-27T09:00:00-00:00
resolved: false
# resolvedWhen: 2021-06-17T09:45:00-00:00
# Possible severity levels: down, disrupted, notice
severity: notice
affected:
  - accounts.jenkins.io
  - beta.accounts.jenkins.io
  - customize.jenkins.io
  - get.jenkins.io
  - incrementals.jenkins.io
  - infra.ci.jenkins.io
  - javadoc.jenkins.io
  - jenkins-wiki-exporter.jenkins.io
  - ldap.jenkins.io
  - plugins.jenkins.io
  - release.ci.jenkins.io
  - reports.jenkins.io
  - www.jenkins.io
  - uplink.jenkins.io
section: issue
---

We'll do an AKS clusters upgrade from 1.20 to 1.21 this Wednesday, 27th of April 2022, starting at 9:00 UTC.
```

**Comment by dduportal** [comment]
Weird: the preview site does not show the notice. Gotta check locally

**Comment by lemeurherve** [comment]
The file extension was missing, all good now: https://deploy-preview-149--pedantic-kepler-514f4a.netlify.app/

**Review by dduportal — APPROVED** [review]
LGTM! Last preview seems to work well and show the notice.

You may proceed when you want!

**Pull request merged**